spark/core/build.gradle

```diff
@@ -0,0 +1,173 @@
+import org.elasticsearch.hadoop.gradle.scala.SparkVariantPlugin
+
+description = "Elasticsearch Spark Core"
+
+apply plugin: 'java-library'
+apply plugin: 'scala'
+apply plugin: 'es.hadoop.build'
+apply plugin: 'spark.variants'
+
+sparkVariants {
+    capabilityGroup 'org.elasticsearch.spark.variant'
+    setDefaultVariant "spark20scala211", spark24Version, scala211Version
+    addFeatureVariant "spark20scala210", spark22Version, scala210Version
+    addFeatureVariant "spark13scala211", spark13Version, scala211Version
+    addFeatureVariant "spark13scala210", spark13Version, scala210Version
+
+    all { SparkVariantPlugin.SparkVariant variant ->
+
+        String scalaCompileTaskName = project.sourceSets
+                .getByName(variant.getSourceSetName("main"))
+                .getCompileTaskName("scala")
+
+        project.configurations {
+            create(variant.configuration('embedded')) {
+                transitive = false
+                canBeResolved = true
```

Comment:
do you need this to be resolved directly? aren't other configurations not just extending this one?